.github/workflows/lints.yml

```diff
@@ -2,7 +2,13 @@ on: [push, pull_request]
 name: Lints on nightly toolchain
 jobs:
   clippy:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
```

Comment:
Can you also add `fail-fast: false` as a child property of the strategy, so that other items in the matrix aren't cancelled when one of them fails?